gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.3.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.5-all.zip
```

Comment:
@larsgrefer what is your reasoning for moving to `-all` from `-bin`?

Reply:
With the `all` distribution, my IDE (IntelliJ IDEA) can provide better auto-completion and documentation for build scripts. Therefore I have `all` as default on my systems.

Reply:
@jzheaux Should I change it back to `bin`?

Reply:
Yes, let's please change it back to `bin` since this ticket is just about updating to Gradle 5.5.

Reply:
Done